scipy/stats/tests/test_multivariate.py

```diff
@@ -3252,22 +3252,42 @@ def test_input_validation(self):
         with assert_raises(ValueError, match=text):
             dirichlet_multinomial.logpmf(x0, [3, -1, 4], n0)
 
-        text = "`alpha` must be an array with exactly one dimension"
-        with assert_raises(ValueError, match=text):
-            dirichlet_multinomial.logpmf(x0, 1, n0)
-
         text = "`n` must be a positive integer."
         with assert_raises(ValueError, match=text):
             dirichlet_multinomial.logpmf(x0, alpha0, 49.1)
         with assert_raises(ValueError, match=text):
             dirichlet_multinomial.logpmf(x0, alpha0, 0)
 
-        text = r"`n` must be an integer \(scalar\)."
-        with assert_raises(ValueError, match=text):
-            dirichlet_multinomial.logpmf(x0, alpha0, [10, 11])
-
         x = np.array([1, 2, 3, 4])
         alpha = np.array([3, 4, 5])
         text = "`x` and `alpha` must be broadcastable."
         with assert_raises(ValueError, match=text):
             dirichlet_multinomial.logpmf(x, alpha, x.sum())
+
+    @pytest.mark.parametrize('method', ['pmf', 'logpmf'])
```

Comment:
I considered parameterizing over all methods, but all thing considered, this is simpler and almost as concise.